### ISSUE TITLE
 Training loss ?

### ISSUE THREAD

**Issue by anujgupta82** [JohnLangford/vowpal_wabbit#1090]
How do i find the training loss ?

```
from vowpalwabbit.sklearn_vw import VWClassifier
model = VWClassifier()
from sklearn import datasets
X, y = datasets.make_hastie_10_2(n_samples=1000, random_state=1)
model.fit(X, y)
```

Is there a way to access the training loss incurred ?

**Comment by JohnLangford** [comment]
Not be default.  If you want training loss, you must deliberately test on the training set.
